btsieve/tests/test_data/find_transaction_if_blockchain_reorganisation/generate_test_data.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+source "../lib.sh"
+
+# Strategy followed to cause a blockchain reorganisation:
+#
+# 1. Start a bitcoin node inside a docker container with a volume.
+# 2. Generate some blocks.
+# 3. Stop the container and save the state by copying it to the file system.
+# 4. Restart the container and generate blocks.
+# 5. Stop the container again and overwrite its current state with the state
+#    saved in step 3.
+# 6. Restart the container and generate blocks. These will be different to
```

Comment:
hum.. ok? I trust you here.

Reply:
Well, hopefully the blockchain is not deterministic :D

Reply:
Well, actually I guess it is deterministic lol.
But since blocks contain timestamps and the time has advances, the blocks will be different.
Also we are including a TX in step 4 so that is also different.

Reply:
I should have used multiline comments :) the comment applies to all the steps.